SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics/Analytics.cs

```diff
@@ -6,11 +6,10 @@
 
 namespace Microsoft.AppCenter.Analytics
 {
-    /// <inheritdoc />
     /// <summary>
     ///     Analytics feature.
     /// </summary>
-    public class Analytics : IAppCenterService
```

Comment:
Fixes crash when adding Analytics nuget to WPF or Winforms (since we don't include dll, portable dll was loaded, implementing an interface but no implementation for ServiceName => TypeLoadException).

Reply:
Works perfectly fine without that by the way, tested direct, and via .NET standard bait/switch (forms.puppet.uwp).